docker/Dockerfile

```diff
@@ -1,14 +1,22 @@
 FROM alpine:3.5
 
+# Enables customized options using environment variables
 ENV BACKEND='http://localhost:5000'
+ENV CENTER='38.8995,-77.0269'
+ENV ZOOM='13'
+ENV LANGUAGE='en'
+ENV LABEL='Car (fastest)'
 
+# Create app directory
 RUN mkdir /src
 COPY . /src
```

Comment:
👍 Sounds like a plan, feel free to change/modify this PR before merging, or close it.